vision/cloud-client/crop_hints/crop_hints.py

```diff
@@ -49,14 +49,14 @@ def get_crop_hint(path):
 
     # Get bounds for the first crop hint using an aspect ratio of 1.77.
     vertices = hints[0].bounding_poly.vertices
-    # [END get_crop_hint]
+    # [END vision_crop_hints_tutorial_get_crop_hints]
 
     return vertices
 
 
 def draw_hint(image_file):
     """Draw a border around the image using the hints in the vector list."""
-    # [START draw_hint]
+    # [START vision_crop_hints_tutorial_draw_crop_hints]
```

Comment:
does the region tag need to be consistent with the function name?  here the tag says `...draw_crop_hints` while the function's name is `draw_hint`.

Reply:
The region tag does not need to have the same name as the function